sdk/core/core-client-rest/src/getClient.ts

```diff
@@ -41,6 +41,17 @@ export function getClient(
   }
 
   const pipeline = createDefaultPipeline(baseUrl, credentials, clientOptions);
+  if (clientOptions.additionalPolicies?.length) {
+    for (const { policy, position } of clientOptions.additionalPolicies) {
+      // Sign happens after Retry and is commonly needed to occur
+      // before policies that intercept post-retry.
+      const afterPhase = position === "perRetry" ? "Sign" : undefined;
```

Comment:
You'll want to update the dependency on `core-rest-pipeline` to have a minimum of `^1.5.0` to support "Sign"